src/sentry/models/rawevent.py

```diff
@@ -22,7 +22,7 @@ class RawEvent(Model):
         wrapper=CanonicalKeyView,
     )
 
-    objects = EventManager()
+    objects = BaseManager()
```

Comment:
Does anyone use RawEvent.objects ?

Reply:
Yes, it's a reasonable way to query with RawEvent since that's a Django model